test/e2e/Deploy.t.sol

```diff
@@ -546,7 +548,8 @@ contract E2EDeploymentMainnetTest is DeployMainnet, CommonDeploymentTest {
   }
 }
 
-contract MainnetDeploymentTest is MainnetDeployment, CommonDeploymentTest {
+// rm "abstract" to reactivate after Deployment
+abstract contract MainnetDeploymentTest is MainnetDeployment, CommonDeploymentTest {
```

Comment:
What about adding `--nmc MainnetDeploymentTest` to the yarn command?

Reply:
hmmm, equivalent